data-structures/linked-list.md

```diff
@@ -5,50 +5,146 @@
+ - [Javascript Arrays](https://github.com/Techtonica/curriculum/edit/master/data-structures/linked-list.md)
+ - [Javascript Objects](https://github.com/Techtonica/curriculum/edit/master/data-structures/linked-list.md)
+ - [Prototypical Inheritance](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Inheritance_and_the_prototype_chain)
 
 ### Motivation
 Though on the job, you will rarely (if ever) asked to implement a data structure from scratch, having a solid knowledge and understanding of the various data structures and ideal use cases can help you ace a technical interview (and get into your dream job working in tech).
 
 ### Objective
-**Apprentices will be able to** implement a linked list.
+**Apprentices will be able to** implement various different types of linked lists. They should understand which portions of linked lists are already implemented in javascript and how to implement their own linked lists under the correct circumstances.
 
-### Specific Things To Teach
-- What is a data structure
+### Specific Things To Learn
+- What is a linked list
 - What are the basic characteristics of a linked list
+  - Why would a linked-list be used instead of an array?
+  - What other data structures are similar to linked lists?
   - What is the difference between a singly-linked list and a doubly linked-list
-- When would a linked-list be used?
-
-### Materials
+  - Why would a singly linked-list be used instead of a doubly linked list?
+- How to recognize linked lists when you see them
+- How to know when to use linked lists to solve a problem.
+- Linked lists in different versions of javascript.
 
+### Supplemental Materials
 - [MDN web docs | Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes)
 - [THat JS Dude | JS: Interview Questions part -4: Stack, Queue, Linked List](https://www.thatjsdude.com/interview/linkedList.html#singlyLinkedList)
 - [Geeks for Geeks | Linked List | Set 1 (Introduction)](https://www.geeksforgeeks.org/linked-list-set-1-introduction/)
+- [ES6 Implementation](https://gist.github.com/klugjo/a9e9ef98fe879bc2b19b5a2e5947204c)
+- [Instantiation Patterns in Javascript](https://medium.com/dailyjs/instantiation-patterns-in-javascript-8fdcf69e8f9b)
 
 ### Mini Lesson
 
 Create a file named "node.js" and create a Node class like the one below but give each Node a 'text' attribute.
 
 <code>
-export default class Node {
-  constructor (){
-  
-  }
- }
-</code>
-
-Next, create a file named "linkedlist.js" and create a LinkedList class object like the following but add any attributes that you deem necessary to create a linked list (based on your research (see links above)).
-<code>
-  const Node = require('./node.js');
-  export default class LinkedList {
-    constructor(head, text){
-    this.head = head || new Node(text)0
-    
-  }
-}  
+// Declare a Node() function that we will use to instantiate new Node objects.
```

Comment:
can you put this in triple backticks with a `js` hint to get syntax hightlighting instead of `<code>` (```js)
```js
let x = 5;
```

Reply:
It also looks there are just generally some markdown formatting issues with the code

Reply:
I'll go through the markdown and submit another PR with any changes I find.

Reply:
I think you should just be able to update the PR here